eZ/Publish/Core/Persistence/Cache/BookmarkHandler.php

```diff
@@ -50,7 +57,7 @@ public function loadByUserIdAndLocationId(int $userId, array $locationIds): arra
     {
         return $this->getMultipleCacheItems(
             $locationIds,
-            'ez-bookmark-' . $userId . '-',
+            $this->cacheIdentifierGenerator->generateKey(self::BOOKMARK_IDENTIFIER, [$userId], true) . '-',
```

Comment:
both `ez-bookmark-` and `bookmark-`  may use `self::BOOKMARK_IDENTIFIER`  now?

Reply:
@konradoboza

Reply:
@vidarl not sure I get your question - the third parameter of `generateKey` method decides whether we add a prefix (`ez-`) to the key itself, so nothing was changed in this line actually.